docs/scripts/build-changelog.mjs

```diff
@@ -1,8 +1,9 @@
+import { Octokit } from '@octokit/core';
 import path from 'path';
-import { simpleGit } from 'simple-git';
 import { fs, globby } from 'zx';
 
-const git = simpleGit();
+require('dotenv').config();
```

Comment:
Maybe change to also use `.env.local` so we don't have multiple env files?

Reply:
(Or does this not work in vercel then?)

Reply:
For me it doesn't work with the local env file

Reply:
But why? You can specify a different .env file for the lib.

Reply:
my token is than undefined

Reply:
how can I test it in vercel? I added a env variable in vercel for it. And the env file from me is created in docs/.env

Reply:
https://marigold-docs-3meutb371-marigold.vercel.app/releases/packages/components/release?theme=core

Reply:
https://www.npmjs.com/package/dotenv#path

```
require('dotenv').config({ path: ['.env.local', '.env'] })
```

Reply:
I just have one env file.....

Reply:
By default, config will look for a file called .env in the current working directory.